index.js

```diff
@@ -53,6 +53,26 @@ module.exports = function postcssPrefixSelector(options) {
   };
 };
 
+function ignoreFile(file, ignoreArr) {
```

Comment:
I'd like to change the name of this function to be more descriptive of what it does, ex: `isFileIgnored `